fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -854,10 +851,7 @@ private void addLinkAndOptionsHttpHeaders() {
             options = "MOVE,COPY,DELETE,POST,HEAD,GET,PUT,PATCH,OPTIONS";
```

Comment:
Should the `else if` check be expanded to also include "|| resource instanceof FedoraTimeMap"?
...like https://github.com/fcrepo4/fcrepo4/pull/1364/files#diff-d372035f89c8d0846920e20bd13e0145R597